Source/Request.swift

```diff
@@ -229,19 +239,19 @@ public class Request {
     /// Default initializer for the `Request` superclass.
     ///
     /// - Parameters:
-    ///   - id:                 `UUID` used for the `Hashable` and `Equatable` implementations. Defaults to a random `UUID`.
+    ///   - id:                 `UUID` used for the `Hashable` and `Equatable` implementations. `UUID()` by default.
     ///   - underlyingQueue:    `DispatchQueue` on which all internal `Request` work is performed.
-    ///   - serializationQueue: `DispatchQueue` on which all serialization work is performed. Targets the
-    ///                         `underlyingQueue` when created by a `SessionManager`.
-    ///   - eventMonitor:       `EventMonitor` used for event callbacks from internal `Request` actions.
+    ///   - serializationQueue: `DispatchQueue` on which all serialization work is performed. By default targets
+    ///                         `underlyingQueue`, but can be passed another queue from a `Session`.
+    ///   - eventMonitor:       `EventMonitor` called for event callbacks from internal `Request` actions.
     ///   - interceptor:        `RequestInterceptor` used throughout the request lifecycle.
     ///   - delegate:           `RequestDelegate` that provides an interface to actions not performed by the `Request`.
-    public init(id: UUID = UUID(),
-                underlyingQueue: DispatchQueue,
-                serializationQueue: DispatchQueue,
-                eventMonitor: EventMonitor?,
-                interceptor: RequestInterceptor?,
-                delegate: RequestDelegate) {
+    init(id: UUID = UUID(),
```

Comment:
Yes, it isn't useful to be able to create a `Request` instance, as it does nothing on its own. Since the subclasses made their initializers non-public as well, this seemed like a good thing to do.